modules/local/rename_raw_data_files.nf

```diff
@@ -25,7 +25,11 @@ process RENAME_RAW_DATA_FILES {
     // Add soft-links to original FastQs for consistent naming in pipeline
     if (meta.single_end) {
         """
-        [ ! -f  ${meta.id}.fastq.gz ] && ln -s $reads ${meta.id}.fastq.gz
+        if [ ! -f  ${meta.id}.fastq.gz ]; then
+            ln -s $reads ${meta.id}.fastq.gz
+        else
+            touch ${meta.id}.fastq.gz
```

Comment:
Otherwise, since the input file is unchanged it's not seen as output. I know it should, since `includeInputs: true` is specified, but it doesn't happen. And, if it happened, the input file would get passed together with the link when names do not collide. Only thing I could think of.

Reply:
Alright, if this appears more stable.